docs.js

```diff
@@ -147,21 +146,11 @@ async function traverseDocsList(result) {
       for (const doc of item.docs) {
         const {repo, repoUrl, docs} = item;
         let {version, commitId} = doc;
-        let date;
```

Comment:
I removed the `date` as it seems to be unimportant in doc but requires many API calls to GitHub

Reply:
The idea is mostly for `latest` version.